### PR TITLE
Paywu/update call answer permissions

### PR DESCRIPTION
**Pull request by paywuAtMicrosoft**

### CODE REVIEW

File: api-reference/beta/api/call-answer.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ The bot is expected to answer or [reject](./call-reject.md) the call before the
...
 
 | Permission type | Permissions (from least to most privileged)                 |
 | :-------------- | :-----------------------------------------------------------|
 | Delegated (work or school account)     | Not Supported                        |
 | Delegated (personal Microsoft account) | Not Supported                        |
-| Application     | Calls.JoinGroupCalls.All or Calls.JoinGroupCallsasGuest.All                                                         |
+| Application     | Calls.JoinGroupCalls.All or Calls.JoinGroupCallsasGuest.All |
 
-> **Note:** For a call that uses application-hosted media, you also need the Calls.AccessMedia.All permission.                                                   |
+> **Note:** For a call that uses application-hosted media, you also need the `Calls.AccessMedia.All` permission. If you don't have any permissions, the `source` in the incoming call notification will be encrypted. To get decrypted `source` in the incoming call notification, you need at least one of the following permissions: `Calls.AccessMedia.All`, `Calls.Initiate.All`, `Calls.InitiateGroupCall.All`, `Calls.JoinGroupCall.All`, `Calls.JoinGroupCallAsGuest.All`.
```
Comment (ananmishr) [inline api-reference/beta/api/call-answer.md]: I might rephrase this as: 
You must have at least one of the following permissions to ensure that the `source` in the incoming call notification is decrypted : `Calls.AccessMedia.All`, `Calls.Initiate.All`, `Calls.InitiateGroupCall.All`, `Calls.JoinGroupCall.All`, `Calls.JoinGroupCallAsGuest.All`.

Without at least one of these permissions, the `source` will remain encrypted.

Comment (ananmishr) [reply api-reference/beta/api/call-answer.md]: We could also add a quick recap to what the source is in the text

**Comment by Lauragra** [comment]
@paywuAtMicrosoft , can you incorporate @ananmishr feedback on this update before we merge? Thanks.

File: api-reference/beta/api/call-answer.md (hunk 1 above)
Comment (paywuAtMicrosoft) [reply api-reference/beta/api/call-answer.md]: updated

Comment (paywuAtMicrosoft) [reply api-reference/beta/api/call-answer.md]: updated

**Comment by paywuAtMicrosoft** [comment]
> @paywuAtMicrosoft , can you incorporate @ananmishr feedback on this update before we merge? Thanks.

Thanks Laura and Ananya. This is done
